google/oauth2/id_token.py

```diff
@@ -12,7 +12,52 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Google ID Token helpers."""
+"""Google ID Token helpers.
+
+Provides support for verifying OAuth 2.0 ID Tokens, especially ones generated
```

Comment:
This are actual OpenID Connect ID Token. Might be worth linking to http://openid.net/specs/openid-connect-core-1_0.html#IDToken

Reply:
Thanks, that's an important terminology distinction and useful link